packages/babel-plugin-transform-async-to-promises/src/index.js

```diff
@@ -0,0 +1,112 @@
+/* Babel 7 transform */
+
+const babylon = require("babylon");
+const generate = require("@babel/generator").default;
+
+function printNode(ast) {
+  return generate(ast, {}).code;
+}
+
+function parse(code) {
```

Comment:
I wondering why you need to parse code here? Isn't it a regular Babel plugin?

Reply:
There is an internal templating system which needs parse the templates.